File: IT/1/Appendici/Moduli_Package_Namespace/Java/package/TestPackage.java
/* file TestPackage.java */

package mypackage;

class Foo {
	public void bar() {
		System.out.println("bar baz");
	}
}

public class TestPackage {
	public static void main(String [] args) {
		Foo foo1 = new Foo();
		foo1.bar();

		mypackage.utils.Foo foo2 = new mypackage.utils.Foo();
		foo2.bar();
	}
}

File: IT/1/Appendici/Moduli_Package_Namespace/Python/moduli/test_package.py
# modulo test_package.py

from __future__ import print_function

import mypackage.utils
# from mypackage.utils import Foo

class Foo:
	def bar(self):
		print("bar baz")

foo1 = Foo()
foo1.bar()

foo2 = mypackage.utils.Foo()
foo2.bar()

